assets/js/qr_reading.js

```diff
@@ -3,8 +3,8 @@ import { Html5Qrcode, Html5QrcodeSupportedFormats } from "../vendor/html5-qrcode
 function parseURL(url) {
   try {
     const url_obj = new URL(url);
-    if (url_obj.host !== window.location.host) return null;
-    return url_obj.pathname.split("/").at(-1);
+    //if (url_obj.host !== window.location.host) return null;
```

Comment:
Remove this commented line, please

Reply:
The commented line is to prevent scanning urls from different hostnames. It's commented because I was running some tests and forgot to uncomment when committing :sweat_smile: Fixed now